content/Hello yaaf - Juan Manuel Ramallo.md

```diff
@@ -0,0 +1,73 @@
+## Yet another active form
+
+Introducing [yaaf](https://github.com/rootstrap/yaaf), a gem to ease the usage of the form object pattern in rails apps.
+
+### ~The~ My pain points
```

Comment:
This section is titled _My pain points_, however in the content you are always referring to _we_. I think the title implies that this is some sort of personal story, however you are speaking in plural. What about changing the title to _Our pain points_ or changing the redaction to a first person style?

Reply:
That is totally a good point. The thing is want to share what I consider pain point while working with form objects. 

In the paragraph when I say "we", I refer to the rails community:
```txt
The form object pattern is widely used across Rails apps,
and yet we tend to write these form objects [...]
```
I would rather change the paragraph with "I" instead of "we" if you prefer that, but I'd rather leave it as it is. What do you think?

Reply:
Okay, I see. It's fine by me then.